consensus/replay.go

```diff
@@ -299,6 +355,19 @@ func (h *Handshaker) ReplayBlocks(
 		"stateHeight",
 		stateBlockHeight)
 
```

Comment:
Decision whether localSync() called before switch/case below in case edge cases remain